app/models/concerns/image.rb

```diff
@@ -0,0 +1,22 @@
+module Image
+  extend ActiveSupport::Concern
+
+  def validate_image(image, field)
+    return unless image.attached?
+    acceptable_types = ["image/jpeg", "image/png", "image/gif"]
+    unless acceptable_types.include?(image.content_type)
+      return errors.add(field, "must be a JPEG or PNG")
+    end
+  end
+
+  def validate_multiple_images(images, field)
+    return unless images.attached?
+    acceptable_types = ["image/jpeg", "image/png", "image/gif"]
```

Comment:
acceptable_types is still a duplicate, can you make it common?

Reply:
Refactored the code